r-package/grf/man/probability_forest.Rd

```diff
@@ -111,7 +111,7 @@ X.test <- matrix(0, 10, p)
 X.test[, 1] <- seq(-1.5, 1.5, length.out = 10)
 p.hat <- predict(p.forest, X.test, estimate.variance = TRUE)
 
-# Plot the estimated success probabilities with 95 \% confidence bands.
+# Plot the estimated success probabilities with 95 % confidence bands.
```

Comment:
Why did these generated files change? (see causal survival too)   It is not necessary to escape the %  here since it will be rendered as an R comment `# ...` https://grf-labs.github.io/grf/reference/causal_survival_forest.html